src/server/page.ts

```diff
@@ -40,7 +41,7 @@ const clientConfig: ClientConfig = {
 const template = (
   route: ServerSideRoute,
   locale: string,
-  adtractionTag: string,
+  adtractionTag: string | null,
```

Comment:
What's the benefit from typing this as `string | null` instead of making it optional with a `?` (which of course means that the type is really `string | undefined`)? If we would just have it optional there wouldn't be the need to use the nullish coalescing operator on line 152?

Reply:
I'm with you, that's what I did at first :D But TS got mad at me for doing that 🤷‍♂️